app/src/main/java/io/islnd/android/islnd/app/NotificationHelper.java

```diff
@@ -0,0 +1,172 @@
+                values,
+                selection,
+                selectionArgs
+        );
+    }
+
+    public static void cancelNotifications(Context context) {
+        setNotificationsToNotActive(context);
+        NotificationManager notificationManager =
+                (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.cancel(NOTIFICATION_ID);
+    }
+
+    private static List<SpannableStringBuilder> getActiveNotifications(Context context) {
+        List<SpannableStringBuilder> activeNotifications = new ArrayList<>();
+
+        String[] projection = new String[] {
+                IslndContract.NotificationEntry.TABLE_NAME + "." + IslndContract.NotificationEntry.COLUMN_NOTIFICATION_USER_ID,
+                IslndContract.NotificationEntry.TABLE_NAME + "." + IslndContract.NotificationEntry.COLUMN_NOTIFICATION_TYPE
+        };
+
+        Cursor cursor = null;
+        try {
+            cursor = context.getContentResolver().query(
+                    IslndContract.NotificationEntry.CONTENT_URI,
+                    projection,
+                    IslndContract.NotificationEntry.TABLE_NAME + "." + IslndContract.NotificationEntry.COLUMN_ACTIVE + " = ?",
+                    new String[] {Integer.toString(IslndContract.NotificationEntry.ACTIVE)},
+                    IslndContract.NotificationEntry.TABLE_NAME + "." + IslndContract.NotificationEntry.COLUMN_TIMESTAMP + " DESC");
+            while (cursor.moveToNext()) {
+                int userId = cursor.getInt(cursor.getColumnIndex(IslndContract.NotificationEntry.COLUMN_NOTIFICATION_USER_ID));
+                int notificationType = cursor.getInt(cursor.getColumnIndex(IslndContract.NotificationEntry.COLUMN_NOTIFICATION_TYPE));
+                String displayName = DataUtils.getDisplayName(context, userId);
+                activeNotifications.add(buildSpannableNotificationString(
+                        context,
+                        displayName,
+                        notificationType)
+                );
+            }
+        } finally {
+            if (cursor != null) {
+                cursor.close();
+            }
+        }
+
+        return activeNotifications;
+    }
+
+    public static SpannableStringBuilder buildSpannableNotificationString(Context context,
+                                                                  String displayName,
+                                                                  int notificationType) {
+        String contentInfo = "";
+
+        switch (notificationType) {
+            case NotificationType.COMMENT:
+                contentInfo = displayName + " " + context.getString(R.string.notification_comment_content);
+                break;
+            case NotificationType.NEW_FRIEND:
+                contentInfo = displayName + " " + context.getString(R.string.notification_new_friend_content);
+                break;
```

Comment:
We could add a default case here that throws an UnsupportedOperation exception. If we did that we would not have to set `contentInfo` to "". I think it would be good because the app would blow up faster if the notification type was not valid.